apb_devel/writing/reference.adoc

```diff
@@ -557,6 +557,180 @@ etherpad_db_host: "{{ lookup('env','ETHERPAD_DB_HOST') | default('mariadb', true
+*openshift* project, which is a global namespace that
+exposes its images and image streams to be available to any authenticated user
+on the cluster. You can take advantage of this by using the `oc new-app` command
+in the *openshift* project to build your image:
++
+----
+$ oc new-app <path_to_bundle_source> \
+    --name <bundle_name> \
+    -n openshift
+----
++
+After a couple of minutes, you should see your image in the internal registry:
++
+----
+$ oc get images | grep <bundle_name>
+
+sha256:b2dcb4b95e178e9b7ac73e5ee0211080c10b24260f76cfec30b89e74e8ee6742   172.30.1.1:5000/openshift/<bundle_name>@sha256:b2dcb4b95e178e9b7ac73e5ee0211080c10b24260f76cfec30b89e74e8ee6742
+----
+
+. Bootstrap the OAB. This is normally also handled by the `apb push` or `apb
+bootstrap` command. The `apb bootstrap` command is preferable for this step
+because it will also relist the service catalog without having to wait five to
+ten minutes.
++
+If you do not have the `apb` tooling installed, you can run:
++
+----
+ifdef::openshift-enterprise[]
+$ oc get route -n openshift-ansible-service-broker
+endif::[]
+ifdef::openshift-origin[]
+$ oc get route -n ansible-service-broker
+endif::[]
+
+NAME       HOST/PORT                                           PATH      SERVICES   PORT        TERMINATION   WILDCARD
+asb-1338   asb-1338-ansible-service-broker.172.17.0.1.nip.io             asb        port-1338   reencrypt     None
+
+$ curl -H "Authorization: Bearer $(oc whoami -t)" -k -X POST https://asb-1338-ansible-service-broker.172.17.0.1.nip.io/ansible-service-broker/v2/bootstrap                                 
+{
+  "spec_count": 38,
+  "image_count": 109
+}
+----
++
+[NOTE]
+====
+The `oc whoami -t` command should return a token and the authenticated user must
+have permissions as described in
+xref:../../apb_devel/cli_tooling.adoc#apb-devel-cli-install-prereqs-access-permissions[Access Permissions].
+====
+
+. Verify the new APB exists in the OAB. This is normally the functionality of the
+`apb list` command. If you do not have the `apb` tooling installed, you can use
+the route gathered from the previous step and run:
++
+----
+$ curl -H "Authorization: Bearer $(oc whoami -t)" -k https://asb-1338-ansible-service-broker.172.17.0.1.nip.io/ansible-service-broker/v2/catalog
+----
++
+You should see a list of all bootstrapped specs and one that is labeled
+`localregistry-<bundle_name>`. Use `|grep <bundle_name>` to help find it, since
+the output is in JSON.
+
+[[apb-devel-writing-ref-remote-clusters-running-apbs]]
+=== Running APBs
+
+Because of the limitations when working with remote clusters, it may be desired
+for users to want the same functionality as the `apb run` command without having
+to rely on the `apb push` command being successful. This is because `apb run`
+implicitly performs `apb push` first before attempting to provision the
+application. 
+
+In order to work around this:
+
+. First, follow the steps described in
```

Comment:
You can probably drop First, since it is already formatted as step one of a procedure